crates/router/src/core/customers.rs

```diff
@@ -570,7 +573,21 @@ pub async fn delete_customer(
                     )
                     .await
                     .switch()?;
+
+                    if let Some(network_token_ref_id) = pm.network_token_requestor_reference_id {
+                        let _resp = network_tokenization::delete_network_token_from_locker_and_token_service(
```

Comment:
Remove the variable name , if its not being used anywhere

Reply:
if the response is not assigned to a variable name, it will be returned, and that is not expected.